deploy/environments/externals.yaml

```diff
@@ -33,7 +33,7 @@ spack:
     - cli-tools
     - cmake@3.15.7
     - cuda@10.1.243
-    - cuda@11.1.0
+    - cuda@11.0.2
```

Comment:
Just wondering if this is a good opportunity to move to 11.2.2 instead of backwards. ([The upstream recipe already has the URLs/hashes](https://github.com/spack/spack/blob/cb79e98c094d6a07815283cbfa0484d9b18ecc81/var/spack/repos/builtin/packages/cuda/package.py#L26-L37).)

Reply:
I concur.

Reply:
I mentioned the rational  in e82bcfc087baa23d593c0305e57acdc79e85388c:

```
  - According to NVIDIA, don't choose 11.1 either stick to 11.0 or 11.2
  - HPCToolkit doesn't support CUDA 11.2
```
So instead of being bleeding edge (which is not required at the moment), I will choose 11.0.2 for the compatibility with the existing tools. (NVIDIA has changed how they expose certain profiling APIs to other tools in 11.2)

Reply:
OK, I read the description but not the full commit messages 😅.

Reply:
yeah, yesterday it was exhausting journey. So I just pushed a commit and didn't update description...